translations/story/04/1008/004 (その目、逸らさずに).json

```diff
@@ -0,0 +1,641 @@
+            "pathId": -6828081603160065625,
+            "blockIdx": 25
+        },
+        {
+            "jpName": "ダイワスカーレット",
+            "enName": "",
+            "jpText": "どっかのバカが、実力差のある相手にも真っ向から喰らいついて……\r\n序盤からハイペースなレースだったのに、遅れずついて行って。",
+            "enText": "There's an idiot that picked a fight against someone above their \ncaliber. Despite their race having a high pace right from the \nstart, they were able to keep up.",
+            "nextBlock": 27,
+            "pathId": 6238973849427641627,
+            "blockIdx": 26
+        },
+        {
+            "jpName": "ダイワスカーレット",
+            "enName": "",
+            "jpText": "そのくせ、直線で加速までしてみせたんだから……\r\n負けてられないって、思ったのよ。",
+            "enText": "And even then, they were also able to accelerate \nat the straight… That made me feel like \nI can't get left behind either.",
+            "nextBlock": 28,
+            "pathId": 1703952952397991869,
+            "blockIdx": 27
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "っ……！　スカーレット、お前……。",
+            "enText": "…! Scarlet, you…",
+            "nextBlock": 29,
+            "pathId": -4996517846573747807,
+            "blockIdx": 28
+        },
+        {
+            "jpName": "ダイワスカーレット",
+            "enName": "",
+            "jpText": "じゃ。アタシ、トレーニングあるから。",
+            "enText": "Well then. I've got more training to do.",
+            "nextBlock": 30,
+            "pathId": -907071519552993263,
+            "blockIdx": 29
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "なんだよアイツ……負けてられない、って……。\r\nだって、俺なんか……こんな、ダサいのに……！！",
+            "enText": "What's up with her… saying she can't get \nleft behind… Even though I'm this lame!",
+            "nextBlock": 31,
+            "choices": [
+                {
+                    "jpText": "挑戦した君が、カッコよく見えたんだ",
+                    "enText": "She thought you were cool, taking on the challenge",
+                    "nextBlock": 31
+                },
+                {
+                    "jpText": "挑戦した貴方が、カッコよく見えたんだよ",
+                    "enText": "She thought you were cool, taking on the challenge",
+                    "nextBlock": 31
+                }
+            ],
+            "pathId": 4065061614828582342,
+            "blockIdx": 30
+        },
+        {
+            "jpName": "ウオッカ",
+            "enName": "",
+            "jpText": "けどっ！　結局俺は、あきらめちまって……！",
+            "enText": "But…! In the end, I still gave up!",
+            "nextBlock": 32,
+            "choices": [
+                {
+                    "jpText": "大事なのはこれからどうするかじゃないか？",
+                    "enText": "What matters is what you do from here on, don't you think?",
+                    "nextBlock": 32
+                },
+                {
+                    "jpText": "大事なのはこれからどうするかじゃない？",
+                    "enText": "What matters is what you do from here on, don't you think?",
```

Comment:
Could you add a newline here or shorten them?
It _just_ doesn't fit the screen (overflow will happen regardless)